components/benefit_card_header.js

```diff
@@ -22,6 +23,9 @@ const parentIcon = css`
 const headerDesc = css`
   flex-grow: 1;
   color: #434343;
+  @media only screen and (max-width: ${globalTheme.max.mobile}) {
+    font-size: 12px;
```

Comment:
You could also apply this to benefit_card_additional_info.js.

We really should combine those 2 components and just send in different text, since they duplicate a lot of stuff.